tests/find_test_under_cursor_test.py

```diff
@@ -88,6 +88,12 @@ class InstanceMethodsTest(TestBase):
     def testStubInstancesInsteadOfClasses(self):
         max = Dog()""",
         'InstanceMethodsTest::testStubInstancesInsteadOfClasses'),
+
+    ("""
+class TestClassEndingWithTests(TestBase):
```

Comment:
```suggestion
class ClassEndingWithTests(TestBase):
```

if the name already starts with "Test" we don't need to change the implementation

Reply:
Excellent point.